front/app/components/ProjectAndFolderCards/index.tsx

```diff
@@ -46,6 +49,13 @@ const ProjectAndFolderCards = ({
     setCurrentTab((currentTab) => getCurrentTab(counts, currentTab));
   }, [counts, currentTab]);
 
+  const handleSearchChange = (search: string | null) => {
```

Comment:
Can only be string (onChangeSearch will always pass a string).

Reply:
These should all be `string | null` as the `SearchInput` can pass either value. It wouldn't have broken anything due to the null check on `search` but I've updated the types for accuracy